a/Alice.alice

```diff
@@ -0,0 +1 @@
+"&O@!dlroW ,olleH
```

Comment:
```suggestion
"&O@dlroW olleH
```
Does it need the `,` and `!`

Reply:
#Fixed
Should be like: https://tio.run/##S8zJTE79/18pJacoP1whPycn1UMpRc0/weH/fwA.